### PR TITLE
CIV 3452 CUI Welsh UI Translations 8

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-3451

### Change description ###
Welsh content translation of the following screens: 

- "Add your timeline of events"
- "Have you paid the claimant the amount you admit you owe"

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by laperla11 — APPROVED** [review]

File: src/main/common/form/models/timeLineOfEvents/timelineRow.ts [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 } from '../../validationErrors/errorMessageConstants';
 import {FREE_TEXT_MAX_LENGTH} from '../../validators/validationConstraints';
 
 export default class TimelineRow {
   @ValidateIf(o => o.isAtLeastOneFieldPopulated())
-  @IsDefined({message: DATE_REQUIRED})
-  @IsNotEmpty({message: DATE_REQUIRED})
+  @IsDefined({message: 'ERRORS.DATE_REQUIRED'})
+  @IsNotEmpty({message: 'ERRORS.DATE_REQUIRED'})
   @MaxLength(FREE_TEXT_MAX_LENGTH, {message: VALID_FREE_TEXT_DATE_LENGTH})
```
Comment (commitBlob) [inline src/main/common/form/models/timeLineOfEvents/timelineRow.ts:12]: why is this not translated?

File: src/main/common/form/models/timeLineOfEvents/timelineRow.ts [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
     date?: string;
 
   @ValidateIf(o => o.isAtLeastOneFieldPopulated())
-  @IsDefined({message: DESCRIPTION_REQUIRED})
-  @IsNotEmpty({message: DESCRIPTION_REQUIRED})
+  @IsDefined({message: 'ERRORS.DESCRIPTION_REQUIRED'})
+  @IsNotEmpty({message: 'ERRORS.DESCRIPTION_REQUIRED'})
   @MaxLength(FREE_TEXT_MAX_LENGTH, {message: VALID_TEXT_LENGTH})
```
Comment (commitBlob) [inline src/main/common/form/models/timeLineOfEvents/timelineRow.ts:18]: why is this not translated?

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -267,7 +279,7 @@
       "RESPONSE_TITLE": "Eich ymateb i'r hawliad",
       "RESPONSE_CHECKBOX": "I believe that the facts stated in this response are true.",
       "RESPONSE_CHECKBOX_PROCEEDINGS": "I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in its truth.",
-      "RESPONSE_HAVE_YOU_PAID_THE_CLAIMANT": "Ydych chi wedi talu'r swm sy'n ddyledus gennych i'r hawlydd?",
+      "RESPONSE_HAVE_YOU_PAID_THE_CLAIMANT": "A ydych wedi talu’r hawlydd y swm rydych chi’n cyfaddef sy’n ddyledus gennych?",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:282]: Has previous translation been marked as incorrect?

File: src/main/modules/i18n/locales/cy.json [hunk 4]
```diff
@@ -403,7 +415,7 @@
       "REASON_FOR_CLAIM": "Reason for claim",
       "TIMELINE": "Timeline",
       "VIEW_AMOUNT_BREAKDOWN": "View amount breakdown",
-      "WHAT_HAPPENED": "What happened"
+      "WHAT_HAPPENED": "Beth ddigwyddodd"
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:418]: Has previous translation been marked as incorrect?

File: src/main/modules/i18n/locales/cy.json [hunk 5]
```diff
@@ -9,16 +9,28 @@
   "View": "Weiv",
   "Save and continue": "Cadw a Pharhau",
   "COMMON": {
+    "APRIL": "Ebrill",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:12]: Personally I would put these into common.months group rather then keeping them all in common

File: src/main/views/features/response/admission/partialAdmission/already-paid.njk [hunk 6]
```diff
@@ -17,20 +17,20 @@
         {% set errors = form.getErrors() %}
         {% if form.hasErrors() %}
           {{ govukErrorSummary({
-            'titleText': 'There was a problem',
-            'errorList': errors
+            titleText: t('ERRORS.THERE_WAS_A_PROBLEM'),
```
Comment (commitBlob) [inline src/main/views/features/response/admission/partialAdmission/already-paid.njk:20]: can we use error summary macro please

File: src/main/modules/i18n/locales/cy.json (hunk 4 above)
Comment (hmcts-version1-domini) [reply src/main/modules/i18n/locales/cy.json:418]: If the sentence within the CY file is in English what I understand is that this sentence wasn't translated before, so there isn't a previous translation.

Comment (commitBlob) [reply src/main/modules/i18n/locales/cy.json:420]: sorry, I swear I've seen something else :D

**Review by commitBlob — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 3 above)
Comment (hmcts-version1-domini) [reply src/main/modules/i18n/locales/cy.json:284]: I have been confirming with some people about it, and have been told to consider the excel doc attached to my task as the proper translation in this case.

**Pull request closed without merging**